pkg/kubectl/cmd/get.go

```diff
@@ -188,12 +193,24 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		}
 
 		// print watched changes
-		w, err := r.Watch(rv)
+		w, err := r.Watch("0")
 		if err != nil {
 			return err
 		}
 
 		kubectl.WatchLoop(w, func(e watch.Event) error {
+			accessor, err := meta.Accessor(e.Object)
+			if err != nil {
+				return fmt.Errorf("failed to create metadata accessor: %v", err)
+			}
+			eventrv, err := strconv.Atoi(accessor.GetResourceVersion())
+			if err != nil {
+				return fmt.Errorf("failed to convert resourceVersion string: %v", err)
```

Comment:
Maybe display the value that failed to convert as well? `failed to convert resourceVersion string %q to an int: %v` or something like that?